packages/core/src/lib/containers.ts

```diff
@@ -49,11 +46,6 @@ export async function getContainer(
     throw Error(`Remote container options is empty`);
   }
 
-  if (typeof remoteContainer === 'string' && globalScope[remoteContainer]) {
```

Comment:
I'm not sure if this code is required or not, this was a rough port from utilities. I think this isn't required going forward though it may be a backwards compatibility thing? I'm not sure if this is node related or the string template "promise new promise" support, @ScriptedAlchemy - would you be able to weigh in there?

Reply:
This code actually is duplicate, a getConteiner key returns the key itself if it's a string